### PR TITLE
FEATURE: SD-569 call afterRender in PivotTable

### PR DESCRIPTION
**Pull request by naibinh**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)(../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Squash commits

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2386
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2196

# Related Jira tasks
- SD-569: https://jira.intgdc.com/browse/SD-569

### CODE REVIEW

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4469/ : SUCCESS in 5m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5742/ : SUCCESS in 7s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2399/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1805/ : SUCCESS in 13m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5743/ : SUCCESS in 12s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1110.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1806/ : SUCCESS in 18m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5744/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4473/ : SUCCESS in 9m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5747/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4478/ : SUCCESS in 8m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5751/ : SUCCESS in 8s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1807/ : SUCCESS in 13m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5752/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4483/ : SUCCESS in 12m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5756/ : SUCCESS in 12s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1809/ : FAILURE in 9m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5759/ : SUCCESS in 7s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4485/ : SUCCESS in 8m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5760/ : SUCCESS in 6s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2406/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1810/ : SUCCESS in 13m 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5761/ : SUCCESS in 8s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1110.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1811/ : FAILURE in 27m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5772/ : SUCCESS in 7s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1110.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1812/ : SUCCESS in 17m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5773/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4501/ : SUCCESS in 8m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5774/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4503/ : SUCCESS in 26m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5775/ : SUCCESS in 13s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4517/ : SUCCESS in 7m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5789/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4545/ : SUCCESS in 7m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5818/ : SUCCESS in 7s

File: src/components/core/PivotTable.tsx [hunk 1]
```diff
@@ -414,6 +420,51 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
...
+        // table has data and takes place of Loading icon
+        if (!this.isTableHidden()) {
+            this.stopWatchingTableRendered();
+            setTimeout(this.props.afterRender, 100);
+        }
+    };
+
+    private stopWatchingTableRendered = () => {
+        clearInterval(this.watchingIntervalId);
+        this.watchingIntervalId = null;
```
Comment (ptquang86) [inline src/components/core/PivotTable.tsx]: should we clear `watchingTimeoutId` also?

File: src/components/core/PivotTable.tsx [hunk 2]
```diff
@@ -414,6 +420,51 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
...
+        // table having no data will be unmounted, it causes ref null
+        if (!this.containerRef) {
+            this.stopWatchingTableRendered();
+            return;
+        }
+
+        // table has data and takes place of Loading icon
+        if (!this.isTableHidden()) {
+            this.stopWatchingTableRendered();
+            setTimeout(this.props.afterRender, 100);
```
Comment (lanhq) [inline src/components/core/PivotTable.tsx]: It seem that it want to waiting for something finished but don't know what. Please consider some how to get rid of this time out.

File: src/components/core/PivotTable.tsx [hunk 3]
```diff
@@ -149,6 +152,9 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
     private lastScrollTop: number = 0;
     private lastScrollLeft: number = 0;
 
+    private watchingIntervalId: number = null;
+    private watchingTimeoutId: number = null;
```
Comment (imnutz) [inline src/components/core/PivotTable.tsx]: Can we remove the default values? The variables have type of number, but have ```null``` as default feels awkward for me.

File: src/components/core/PivotTable.tsx [hunk 4]
```diff
@@ -414,6 +420,51 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         }
     };
 
+    private startWatchingTableRendered = () => {
+        // table having no data will be unmounted, it causes ref null
+        if (!this.containerRef) {
+            this.stopWatchingTableRendered();
+            return;
```
[the remark this answers is not in the document]
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: I move calling `afterRender` to `stopWatchingTableRendered`.

File: src/components/core/PivotTable.tsx (hunk 1 above)
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: I will be cleaned in `watchingTableRenderedTimeout`

File: src/components/core/PivotTable.tsx (hunk 3 above)
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: Done

File: src/components/core/PivotTable.tsx (hunk 2 above)
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: Test and removed timeout.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4547/ : SUCCESS in 4m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5822/ : SUCCESS in 6s

File: src/components/core/PivotTable.tsx [hunk 5]
```diff
@@ -414,6 +420,51 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
...
+            this.props.afterRender();
+        }
+        clearTimeout(this.watchingTimeoutId);
+        this.watchingTimeoutId = null;
+    };
+
+    private onFirstDataRendered = () => {
+        // Since issue here is not resolved, https://github.com/ag-grid/ag-grid/issues/3263,
+        // work-around by using 'setInterval'
+        this.watchingIntervalId = window.setInterval(
```
[the remark this answers is not in the document]
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: `setInterval / setTimeout` will reture `NodeJS.Timer`.
Only under `window` object, they return `number`. 
More detail [here](https://stackoverflow.com/questions/45802988/typescript-use-correct-version-of-settimeout-node-vs-window).

`clearTimeout` and `clearInterval` accept `number` naturally.
So I think we don't have to call them under `window` strictly.

**Comment by naibinh** [comment]
extended test - storybook

File: src/components/core/PivotTable.tsx (hunk 1 above)
Comment (ptquang86) [reply src/components/core/PivotTable.tsx]: In this case, after `afterRender` is called, `watchingTableRenderedTimeout` will be called also when `WATCHING_TABLE_RENDERED_MAX_TIME` is met.

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2431/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1827/ : SUCCESS in 13m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5823/ : SUCCESS in 11s

File: src/components/core/PivotTable.tsx (hunk 1 above)
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: Fixed.
You're right, we should clean timeout ASAP.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4549/ : SUCCESS in 5m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5825/ : SUCCESS in 7s

**Comment by naibinh** [comment]
extended test - deploy - backend staging3.intgdc.com

**Comment by yenkins** [comment]
Demo is running at https://client-demo.na.intgdc.com:50038/storybook/

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1829/ : SUCCESS in 8m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5827/ : SUCCESS in 8s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1110.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1830/ : FAILURE in 49m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5828/ : SUCCESS in 10s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1110.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1834/ : SUCCESS in 9m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5836/ : SUCCESS in 7s

**Comment by ptquang86** [comment]
LGTM

**Comment by strouhaljan** [comment]
Hello guys, sorry this takes so long, but we are currently discussing the solution with our architect, because of the uncommon approach. Ag-grid is not helping us with this problem and we are aware there was no other way, but using `setInterval`s.

File: src/components/core/PivotTable.tsx [hunk 6]
```diff
@@ -149,6 +152,9 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
     private lastScrollTop: number = 0;
     private lastScrollLeft: number = 0;
 
+    private watchingIntervalId: number;
```
[the remark this answers is not in the document]
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: Yes, I fixed it.

File: src/components/core/PivotTable.tsx [hunk 7]
```diff
@@ -414,6 +420,49 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
...
+
+        // table has data and takes place of Loading icon
+        if (!this.isTableHidden()) {
+            this.stopWatchingTableRendered();
+        }
+    };
+
+    private stopWatchingTableRendered = () => {
+        if (this.watchingIntervalId) {
+            clearInterval(this.watchingIntervalId);
```
[the remark this answers is not in the document]
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: Great, it's more meaningful now.
Fixed.

File: src/components/core/PivotTable.tsx [hunk 8]
```diff
@@ -414,6 +420,49 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         }
     };
 
+    private startWatchingTableRendered = () => {
+        // table having no data will be unmounted, it causes ref null
+        if (!this.containerRef) {
```
[the remark this answers is not in the document]
Comment (naibinh) [reply src/components/core/PivotTable.tsx]: Great, it's more meaningful now.
Fixed.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4580/ : SUCCESS in 13m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5869/ : SUCCESS in 16s

**Comment by naibinh** [comment]
@strouhaljan, please check my latest commit.

**Comment by strouhaljan** [comment]
@naibinh looks good to me now :-) Feel free to squash all the commits so we can merge it.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2449/ : SUCCESS in 13m 37s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4582/ : SUCCESS in 4m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1157/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1471/ : FAILURE in 21m 08s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1110.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1845/ : FAILURE in 9m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5871/ : SUCCESS in 8s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1110.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4583/ : SUCCESS in 10m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5872/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1846/ : FAILURE in 19m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5873/ : SUCCESS in 9s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1110.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4585/ : SUCCESS in 11m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5875/ : SUCCESS in 12s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1847/ : SUCCESS in 22m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5877/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4587/ : SUCCESS in 7m 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5878/ : SUCCESS in 13s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2450/ : SUCCESS in 10m 41s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4588/ : SUCCESS in 7m 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1158/ : SUCCESS in 57s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1475/ : SUCCESS in 16m 43s

**Pull request merged**